include/sot/torque_control/joint-torque-controller.hh

```diff
@@ -106,19 +108,27 @@ namespace dynamicgraph {
         DECLARE_SIGNAL_IN(motorParameterKa_p, dynamicgraph::Vector);
         DECLARE_SIGNAL_IN(motorParameterKa_n, dynamicgraph::Vector);
         DECLARE_SIGNAL_IN(polySignDq,         dynamicgraph::Vector);
-
+        DECLARE_SIGNAL_IN(torque_integral_saturation, dynamicgraph::Vector);  /// integral error saturation
         DECLARE_SIGNAL_OUT(u,                 dynamicgraph::Vector);  /// Desired current
-        DECLARE_SIGNAL_OUT(smoothSignDq,      dynamicgraph::Vector); /// smooth approximation of sign(dq)
         DECLARE_SIGNAL_OUT(torque_error_integral, dynamicgraph::Vector); /// integral of the torque tracking error
+        DECLARE_SIGNAL_OUT(smoothSignDq,      dynamicgraph::Vector); /// smooth approximation of sign(dq)
+        DECLARE_SIGNAL_INNER(dq_motor,      dynamicgraph::Vector);
```

Comment:
I would declare dq_motor as an output signal so that we can plot it for debugging